app/src/main/java/com/example/atheneum/Book.java

```diff
@@ -282,4 +283,21 @@ public void deletePhoto(String photo) {
     public void deletePhotos() {
         this.photos.clear();
     }
+
+    /**
```

Comment:
When you implement equals() don't you also have to implement hashcode?

Reply:
I think you don't *have* to implement hashcode, though I had planned to leave in the default generated one anyway. Problem was that our Java API level was too low to use Object.hash so I deleted it.